api-datatypes.go

```diff
@@ -82,6 +82,9 @@ type ObjectInfo struct {
 	// x-amz-meta-* headers stripped "x-amz-meta-" prefix containing the first value.
 	UserMetadata StringMap `json:"userMetadata"`
 
+	// x-amz-tagging values in their k/v values.
+	UserTags map[string]string `json:"userTags"`
```

Comment:
This is not being returned as part of get/head object call

Reply:
This is an extension proposal, to be returned as part of get/head call to avoid making another GetObjectTag call @kannappanr to facilitate faster mirror operations.